### PR TITLE
Configure v2 birth and death forms to use new address component configs

### PR DESCRIPTION
**Pull request by pankaj-pant**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: src/form/v2/birth/forms/pages/mother.ts [hunk 1]
```diff
@@ -346,12 +348,23 @@ export const mother = defineFormPage({
           conditional: requireMotherDetails
         }
       ],
+      validation: [
+        {
+          message: {
```
Comment (makelicious) [inline src/form/v2/birth/forms/pages/mother.ts:353]: intentionally generic?

File: src/form/v2/birth/forms/pages/informant.ts [hunk 2]
```diff
@@ -400,12 +402,23 @@ export const informant = defineFormPage({
           conditional: informantOtherThanParent
         }
       ],
+      validation: [
+        {
+          message: {
+            defaultMessage: 'Invalid input',
```
Comment (makelicious) [inline src/form/v2/birth/forms/pages/informant.ts:408]: intentionally generic?

File: src/form/street-address-configuration.ts [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+    required: false,
+    label: {
+      id: 'v2.field.address.addressLine3.label',
+      defaultMessage: 'Address Line 3',
+      description: 'This is the label for the field'
+    },
+    type: FieldType.TEXT
+  },
+  {
+    id: 'postcodeOrZip',
```
Comment (makelicious) [inline src/form/street-address-configuration.ts:228]: should we just call it either post(al) code or zip? In code probably better, even though the label might do that.

File: src/form/street-address-configuration.ts [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+    required: true,
+    label: {
+      id: 'v2.field.address.district2.label',
+      defaultMessage: 'District',
+      description: 'This is the label for the field'
+    },
+    type: FieldType.TEXT
+  },
+  {
+    id: 'cityOrTown',
```
Comment (makelicious) [inline src/form/street-address-configuration.ts:160]: municipality, no need to use or?
